Tests/RespondersTests/RouteResponderTest.swift

```diff
@@ -98,7 +98,7 @@ class RouteResponderTest: XCTestCase {
 
       let expected = "GET /someRoute HTTP/1.1"
 
-      XCTAssertEqual(response.body!, expected.toBytes)
+      XCTAssertEqual(response.body!.toBytes, expected.toBytes)
```

Comment:
The gist of the problem is that I'd like to be able to compare the response body, which is a `BytesRepresentable` type, with these String expectations (BytesRepresentable is currently extended to both `String` and `Data` types) without having to turn them both into `[UInt8]` arrays first.